docs/tutorials/NMR/Example_1_-Ethanol.md

```diff
@@ -0,0 +1,108 @@
+
+
+## Files   
+
+
+*ethanol.cell*
+```
+
+%BLOCK LATTICE_ABC
+6 6 6
+90 90 90
+%ENDBLOCK LATTICE_ABC
+
+%BLOCK POSITIONS_ABS
+H 3.980599 4.178342 3.295079
+H 5.033394 3.43043 4.504759
+H 5.71907 4.552257 3.315353
+H 3.720235 5.329505 5.509909
+H 4.412171 6.433572 4.317001
+H 5.911611 5.032284 6.242202
+C 4.84694 4.350631 3.941136
+C 4.603025 5.518738 4.882532
+O 5.746254 5.812705 5.6871
+%ENDBLOCK POSITIONS_ABS
+
+%BLOCK KPOINTS_LIST
+0.25 0.25 0.25 1.0
+%ENDBLOCK KPOINTS_LIST
+```
+
+*ethanol.param*
+```
+xcfunctional = PBE
+fix_occupancy = true
+opt_strate.g.y : speed
+task        = magres
+
+cut_off_energy  = 20 ry
+```
+
+
+
+## Objectives:
+
+1. Examine the convergence of the chemical shieldings with planewave cutoff.
+2. Compare to experiment.
+
+## Instructions:
+
+1. Look at the [cell](../../documentation/Input_Files/cell_file.md) and [param](../../documentation/Input_Files/param_file.md) files. Note that the only special keyword is `task = magres`
+
+ 2. Run castep
+  `castepsub -n 4  ethanol`
+   Look at the ***ethanol.castep*** output file. At the end, the isotropic chemical shielding, anisotropy, and asymmetry are reported.
+ The result should contain these lines:
+ ```
+ ====================================================================
+ |                      Chemical Shielding Tensor                   |
+ |------------------------------------------------------------------|
+ |     Nucleus                            Shielding tensor          |
+ |  Species            Ion            Iso(ppm)   Aniso(ppm)  Asym   |
+ |    H                1               29.45       8.84      0.14   |
+ |    H                2               30.10       8.07      0.20   |
+ |    H                3               29.94       7.12      0.06   |
+ |    H                4               26.83       8.02      0.95   |
+ |    H                5               27.24      -7.07      0.90   |
+ |    H                6               31.93      13.99      0.46   |
+ |    C                1              157.27      33.77      0.70   |
+ |    C                2              110.73      69.91      0.42   |
+ |    O                1              268.63     -50.78      0.96   |
+ ====================================================================
+ ```
+ Here we are only interested in the isotropic values
+
+
+3. This information, plus the full tensors, are also given in the file ethanol.magres
+
+4. You might wish to transfer the ***ethanol.magres*** file back to your desktop to visualise with [MagresView](https://www.ccpnc.ac.uk/magresview/magresview/magres_view.html?JS).
+
+5. Identify which (hydrogen) ion corresponds to which part of the molecule - in the case of ethanol, find out which ones correspond to CH3~3~ CH~2~ and OH.
+This can be done by uploading the cell file to VESTA and clicking each atom. <br>![NMR vesta demonstration](../img/NMR_vesta_demonstration.png){width="40%"} <br>Here you can see that one of the CH~3~ hydrogens is atom 3
```

Comment:
Can avoid the `<br>` here and elsewhere with markdown indentation.
```suggestion
5. Identify which (hydrogen) ion corresponds to which part of the
   molecule - in the case of ethanol, find out which ones correspond
   to CH3~3~ CH~2~ and OH. This can be done by uploading the cell
   file to VESTA and clicking each atom.

    ![NMR vesta demonstration](../img/NMR_vesta_demonstration.png){width="40%"}

    Here you can see that one of the CH~3~ hydrogens is atom 3
```